egui/src/grid.rs

```diff
@@ -280,6 +428,11 @@ impl Grid {
         self
     }
 
+    pub fn header_row(mut self, header_row: bool) -> Self {
+        self.header_row = header_row;
```

Comment:
This can just add to `color_spec.row_pickers` directly. Same for `striped`

Reply:
The problem is that if someone subsequently calls `header_row(false)`, it's not trivial to figure out how to remove the added row coloration from the `row_pickers`, and also kinda weird to be comparing closures/fn pointers/whatever. Would appreciate thoughts.

Reply:
Perhaps header_row() can be a method with no arguments that, when called, irreversibly sets the header coloration, and same for stripes?